### DISCUSSION TITLE
Welcome to MaterialDesignInXamlToolkit Discussions!

### DISCUSSION THREAD

**Discussion by Keboo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2673] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MichelMichels** [comment]
Hi, I'm @MichelMichels, a Belgian .NET developer. At the company I work, I've been using this style library for over 4 years for our internal software to deliver a more modern WPF application. I try to help with issues on the project and from time to time, I submit PR's.

**Comment by quicoli** [comment]
Hi! I'm @quicoli, a Brazilian living in Ireland. I'm user since its very beginning and was able to bring it to the company I work for. We develop medical devices and bring great UX using MaterialDesignInXamlTookit. Some time ago I collaborated with a 'fix' for a memory leak https://github.com/quicoli/MaterialDesignThemes.CustomResources.
